e2e/react-start/css-modules/tests/css.prod.spec.ts

```
@@ -0,0 +1,80 @@
+    }).toPass({ timeout: 5000 })
+  })
+
+  test('styles work correctly after client-side navigation', async ({
+    page,
+  }) => {
+    await page.goto('/')
+
+    const globalElement = page.getByTestId('global-styled')
+    await expect(globalElement).toBeVisible()
+
+    let backgroundColor = await globalElement.evaluate(
```

ESLint

[error] 'backgroundColor' is never reassigned. Use 'const' instead.

(prefer-const)